prime-router/metadata/fhir_mapping/hl7/secondary/Source.yml

```diff
@@ -136,6 +136,7 @@ extension_3:
     urlValue: software-install-date
 
 extension_4:
+  condition: $sft1 NOT_NULL
```

Comment:
Without this condition we were frequently adding an SFT segment even when it did not exist in the original HL7